pkg/executables/cmk.go

```diff
@@ -338,10 +338,7 @@ func (c *Cmk) ValidateAccountPresent(ctx context.Context, profile string, accoun
 }
 
 // NewCmk initializes CloudMonkey executable to query CloudStack via CLI.
-func NewCmk(executable Executable, writer filewriter.FileWriter, config *decoder.CloudStackExecConfig) (*Cmk, error) {
-	if config == nil {
```

Comment:
This error check might still be needed, need to see if this is the right way to implement.